### ISSUE TITLE
I have a situation related to Booking Please let me know about that.

### ISSUE THREAD

**Issue by abdulrafy32** [dejwid/airbnb-clone#6]
It was great Project. But I have a question related to the Booking functionality. Please let me know about this I am explaining my situation below.
1: Let suppose There is a resort for rent or a stay for few days or weeks RESORT NAME (victoria).
2: There is a user profile "Named : JACK" booked the resort (victoria) for stay for 2 day from 24-5-2023 to 26-5-2023. And  there is Second user profile "Named: ROSE" want to check the availability of the resort on the same dates on which the Jack already booked the resort. Is there any indication for second person or user to check that the place is already booked or not ? 

In short my question is : how should the second user will came to know that the place is already booked or not. there is any functionality for this you added in it.
Looking forward for the replay. Thank you

**Comment by vishal-sanap** [comment]
I think we will have to configure it further with another function that doesn't allow booking for the same dates after one user has already booked it. But it would get a little heavy as we will also have to add user function on the booking page then.

**Comment by abdulrafy32** [comment]
Hi, hope you are doing well.
I got your point and I am trying to figure out how we can achieve this
functionality in an easier way.
Thankyou so much for your response. Have good day
